core/jazz_ui/src/app/pages/environment-overview/env-overview-section.component.ts

```diff
@@ -41,16 +41,19 @@ export class EnvOverviewSectionComponent implements OnInit {
   friendlyName : string;
   yamlName:string;
   lastCommitted: any;
-  isCancel:boolean=false;
   editBtn:boolean = true;
   saveBtn:boolean = false;
-  saveButton:boolean=false;
+  slsapp:boolean = false;
+  showCncl:boolean=false;
   editButton:boolean=true;
+  saveButton:boolean=false;
+  isCancel:boolean=false;
+  version: string = ">=1.0.0 <2.0.0";
+  isvalid:boolean=false;
```

Comment:
consistent camelCase for variables: isValid, startNew?

Reply:
Addressed.